### ISSUE TITLE
Add Group Description

### ISSUE THREAD

**Issue by anguspiv** [opensource-la/opensource.la#2]
Need to add a description of the group and its purpose. Should be added into the ReadMe section.

**Comment by powernerd** [comment]
Opensource.la is:
- [x] A meet up
- [ ] A website
- [x] An initiative to support open source projects
- [x] A place to teach and learn open source technology

**Comment by anguspiv** [comment]
I don't think we need to really mention the website, that's more a communication channel for the organization. 

I think points 3 and 4 should be the main focus of the description
